index.js

```diff
@@ -15,9 +15,33 @@ const isBinary = buffer => {
 	return false;
 };
 
+const cleanEntities = svg => {
+	const entityRegex = /\s*<!Entity\s+\S*\s*(?:"|')[^"]+(?:"|')\s*>/img;
```

Comment:
I think you're over-complicating it. We don't really use the `<!Entity` tag for detecting SVG, so we can just remove the whole tag.

Reply:
Oh, sorry. I just chose to keep the entity tag because sometimes when you define an entity id (Smile, in this example) you can reference it on the svg code and it must exist for the svg to be valid. I'll remove the entities altogether.

Reply:
The regex and code could still be simplified. `.replace` supports a callback you could use instead of both `.match` and `.replace.`.

Reply:
Actually, you can just use normal replace with:

```suggestion
	const entityRegex = /<!Entity[^>]*?>/ig;
```

Reply:
You are right. I replaced match with a single normal replace, but I had to keep my previous regex.
The tests were failing since`/<!Entity[^>]*?>/ig` would only match up until the first markup tag closure. Meaning that if that any subsequent markup tags would not be removed and the match would fail as before.